packages/components/src/navigation/styles/navigation-styles.tsx

```diff
@@ -134,11 +134,11 @@ export const ItemBaseUI = styled.li`
 
 	&.is-active {
 		background-color: ${ COLORS.theme.accent };
-		color: ${ COLORS.white };
+		color: ${ COLORS.theme.accentInverted };
```

Comment:
👍 While this is unchanged on the default theme, this is a nice improvement for the dark one, actually:

|Before|After|
|---|---|
|![Screenshot 2024-12-09 at 11 46 41](https://github.com/user-attachments/assets/4894b4ee-8eb1-449d-8055-0c4501915af9)|![Screenshot 2024-12-09 at 11 46 37](https://github.com/user-attachments/assets/16958ea4-235f-425e-a988-5e8ad8517a72)|